webapp/channels/src/components/dot_menu/dot_menu.tsx

```diff
@@ -358,6 +385,12 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             this.handleDeleteMenuItemActivated(event);
             break;
 
+        // move thread
+        case Keyboard.isKeyPressed(event, Constants.KeyCodes.W):
```

Comment:
Do we want to rethinkg the `W` as hotkey? I understand it comes from Wrangle, but the text says "Move thread", so it is not obvious.